luckystrikehoreca/resources/views/order.blade.php

```diff
@@ -33,46 +33,35 @@
           </tr>
         </thead>
         <tbody id="orderTable">
-          <!-- JavaScript table -->
+          <?php
+          $totalPrice = 0;
+          if (session()->has('orderData')) {
+              $orderData = session('orderData');
+              $decodedOrderData = json_decode($orderData, true);
+              foreach ($decodedOrderData as $item) {
```

Comment:
u can use @foreach here so its not plain php

Reply:
@php
    $totalPrice = 0;
    if (session()->has('orderData')) {
        $orderData = session('orderData');
        $decodedOrderData = json_decode($orderData, true);
    @endphp

    @foreach ($decodedOrderData as $item)
        @php
            $totalItemPrice = $item['price'] * $item['quantity'];
            $totalPrice += $totalItemPrice;
            $formattedPrice = number_format($item['price'], 2);
        @endphp

        <tr>
            <td>{{ $item['item'] }}</td>
            <td>€{{ $formattedPrice }}</td>
            <td>{{ $item['quantity'] }}</td>
            <td class="price">€{{ number_format($totalItemPrice, 2) }}</td>
        </tr>
    @endforeach

@php
    }
@endphp